docs/migration.md

```diff
@@ -0,0 +1,418 @@
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.5.RELEASE</version>
+    <relativePath/>
+</parent>
+
+<dependencies>
+    <dependency>
+        <groupId>com.symphony.platformsolutions</groupId>
+        <artifactId>symphony-api-client-java</artifactId>
+        <version>1.3.3</version>
+    </dependency>      
+</dependencies>
+```
+</td>
+<td>
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>org.finos.symphony.bdk</groupId>
+            <artifactId>symphony-bdk-bom</artifactId>
+            <version>2.1.1</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+
+<dependencies>
+<dependency>
+    <groupId>org.finos.symphony.bdk</groupId>
+    <artifactId>symphony-bdk-core-spring-boot-starter</artifactId>
+</dependency>
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter</artifactId>
+</dependency>
+</dependencies>
+```
+</td>
+</tr>
+</table>
+
+### No framework based project
+
+<table>
+<tr>
+<th>Java SDK</th>
+<th>Java BDK 2.0</th>
+</tr>
+<tr>
+<td>
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>com.symphony.platformsolutions</groupId>
+        <artifactId>symphony-api-client-java</artifactId>
+        <version>1.3.3</version>
+    </dependency>
+</dependencies>
+```
+</td>
+<td>
+
+```xml
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>org.finos.symphony.bdk</groupId>
+            <artifactId>symphony-bdk-bom</artifactId>
+            <version>2.1.1</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+
+<dependencies>
+<dependency>
+    <groupId>org.finos.symphony.bdk</groupId>
+    <artifactId>symphony-bdk-core</artifactId>
+</dependency>
+</dependencies>
+```
+</td>
+</tr>
+</table>
+
+## Bot's configuration
+In Java SDK, two configuration files were required : `application.yaml` (or `application.config`) and `bot-config.json`. Java BDK 2.0 lightened the configuration. Therefore, only `src/main/resources/config.yaml` file is required with a minimum of configuration.
+
+Bot’s configuration in Java BDK 2.0 should have the following properties:
+- `host`: pod’s host name
+
+- `bot.username`: bot’s (or service account) username
+
+- `bot.privatekey.path`: bot’s private key path
+
+If your bot is deployed on premise, the following properties are required as well:
+
+- `agent`: on premise agent configuration
+
+- `keyManager`: on premise Key manager configuration
+
+- `proxy`: proxy configuration to reach the pod
+
+- `ssl.trustStore`: trust store path and password
+
+> Click [here](./configuration.md) for more detailed documentation about BDK configuration
+
+### Minimal configuration example
+#### Spring Boot based project
+<table>
+<tr>
+<th>Java SDK</th>
```

Comment:
This is actually the Java BDK 1.0

Reply:
I replace all "Java SDK" by "Java BDK 1.0" in next commit